cmd/exec.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/okteto/cnd/k8/exec"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/okteto/cnd/k8/client"
+	"github.com/okteto/cnd/model"
+	"github.com/spf13/cobra"
+)
+
+//Exec executes a command on the CND container
+func Exec() *cobra.Command {
+	var devPath string
+	cmd := &cobra.Command{
+		Use:   "exec COMMAND",
+		Short: "Execute a command in the cloud native environment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return executeExec(devPath, args)
+		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return errors.New("exec requires the COMMAND argument")
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVarP(&devPath, "file", "f", "dev.yml", "dev yml file")
+	return cmd
+}
+
+func executeExec(devPath string, args []string) error {
+	namespace, client, config, err := client.Get()
+	if err != nil {
+		return err
+	}
+
+	dev, err := model.ReadDev(devPath)
+	if err != nil {
+		return err
+	}
+
+	pods, err := client.CoreV1().Pods(namespace).List(v1.ListOptions{
+		LabelSelector: fmt.Sprintf("cnd=%s", dev.Name),
```

Comment:
@pchico83 can you think of a better selector for this? This won't be unique once we have more than one environment on the same cluster. 

Maybe we need to update deploy to add the unique pod name to the yaml file?